sdk/storage/azure-storage-files-shares/src/share_directory_client.cpp

```diff
@@ -197,6 +197,118 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
       throw;
     }
   }
+  Azure::Response<ShareFileClient> ShareDirectoryClient::RenameFile(
+      const std::string& fileName,
+      const std::string& destinationFilePath,
+      const RenameFileOptions& options,
+      const Azure::Core::Context& context) const
+  {
+    auto sourceFileUrl = m_shareDirectoryUrl;
+    sourceFileUrl.AppendPath(_internal::UrlEncodePath(fileName));
+
+    auto destinationFileUrl = m_shareDirectoryUrl;
```

Comment:
is absolute path supported?

Like the source is `/dir1/file1`, the destination is `/dir2/file2` or `../dir2/file2`

Reply:
change to absolute path after discussion